org-formation/600-access/_tasks.yaml

```diff
@@ -86,6 +86,36 @@ NlpSandboxServiceAccount:
     Account: !Ref SandboxAccount
     Region: !Ref primaryRegion
 
+# JumpCloud access for Sage staff working on HTAN infrastructure
+JCHtanDevAdmin:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.4/templates/IAM/jumpcloud-idp.yaml
+  StackName: jc-htan-dev-admin
+  MaxConcurrentStacks: 10
+  Parameters:
+    MetadataDocument: !ReadFile ./MetadataDocuments/htan-dev-admin.xml
+    SamlProviderName: htan-dev-admin
+    SamlProviderPolicyArns:
+      - arn:aws:iam::aws:policy/AdministratorAccess
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: false
+    Account: !Ref HtanDevAccount
+    Region: us-east-1
+
+# Service account for provisioner repository: https://github.com/ncihtan/aws-htan-infra
+HtanDevCIServiceAccount:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.4/templates/IAM/service-account.yaml
+  StackName: htan-dev-ci-service-account
+  Parameters:
+    ManagedPolicyArns:
+      - arn:aws:iam::aws:policy/AdministratorAccess
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: false
+    Account:
+      - !Ref HtanDevAccount
+    Region: us-east-1
+
```

Comment:
We've replaced AWS SAML with SSO therefore you don't need to do this part anymore.  Sage IT will setup user access for you.  Please remove.

Reply:
I'll get rid of the JumpCloud access and leave the CI service account.